### ISSUE TITLE
`buffer_data_numpy` should handle bad/empty frames

### ISSUE THREAD

**Issue by morefigs** [morefigs/pymba#106]
Currently it crashes, it should be able to handle them instead.

**Comment by rk-exxec** [comment]
Yes please look into this, I just finished a 2 day jurney to find ot why my Qt Application crashes silenty after stopping the camera feed.
It was the buffer_data_numpy() function inside the frame handle callback. I put the call inside a try-except block and now it works.
